### ISSUE TITLE
SVG Rendering - Gravity Suggestion

### ISSUE THREAD

**Issue by jsavage** [jonasthewolf/gsn2x#173]
Hi @jonasthewolf,
This is a minor observation.
While in general the generated SVG is pleasing to the eye and intuitive, here are a couple of examples where the layout is less intuitive.
![image](https://user-images.githubusercontent.com/189524/194906014-8e957f65-1b1d-4708-8ea2-a0239baae409.png)
Hopefully you can see the issue without being able to read the underlying text. 

I wonder whether gravity might encourage a more pleasing result.

**Comment by jsavage** [comment]
Ah, I have just seen your note that  "The order of the GSN elements on the same horizontal rank can be influenced by their ID. The elements are sorted lexicographically. Thus, a goal G1 if placed on the same vertical level is placed before G2." Will take a closer look at the YAML to see if this is causing the lower of the above issues.

**Comment by jonasthewolf** [comment]
I checked your minimal example:
If you replace the Sn216 with Snx216, the rendering appears as I guess you would expect it.
I am glad you found the place in the documentation, that I tried to explain how to solve such issues.

```
G5:
  text: Rationale
  supportedBy: [Sn227, Sn211]

G8:
  text: Completeness of HSRs
  supportedBy: [Snx216, Sn227]

Snx216:
  text: HSRs

Sn227:
  text: Requirements Traceability Reports
  url: txt

Sn211:
  text: TSRs Report
  url: txt
```

**Comment by jsavage** [comment]
Snx216 didn't but changing Sn211 to Snx211 certainly worked as a workaround.
Some degree of influence from the id seems fine in the absence of other forces but avoiding un-sightly crossing of connections would I am sure be a preferable default. 
![image](https://user-images.githubusercontent.com/189524/195199502-09fac3fc-09da-45f6-a332-8873e1685cdc.png)

**Comment by jonasthewolf** [comment]
Absolutely agreed, @jsavage.

I will check if ordering can be improved.

**Comment by jonasthewolf** [comment]
@jsavage I just merged a small change that should resolve the issue you found without the workaround.
You can find a binary on the releases page (latest pre-release).
Please let me know if you see the issue again. Thanks.

**Comment by jsavage** [comment]
@jonasthewolf  Were you referring to [2.4.3?](https://github.com/jonasthewolf/gsn2x/releases/tag/v2.4.3). If so, when I removed the workaround the issue returned. 
![image](https://user-images.githubusercontent.com/189524/197175413-9971c677-6f41-496a-b548-3a85b81d47a8.png)
[jsexample1.5-db.zip](https://github.com/jonasthewolf/gsn2x/files/9838223/jsexample1.5-db.zip)

**Comment by jonasthewolf** [comment]
Hi @jsavage, I was referring to https://github.com/jonasthewolf/gsn2x/releases/tag/latest-build

**Comment by jsavage** [comment]
Ah yes, got it. Resolved.  The cross-over is resolved with workaround removed. 
![image](https://user-images.githubusercontent.com/189524/197509476-67a3faac-84b9-4170-8314-daa959fe2fb3.png)

**Comment by jonasthewolf** [comment]
@jsavage I released version 2.5.0.
That should improve layout even further.

**Comment by jonasthewolf** [comment]
@jsavage I think with v2.5.1 also the other layout issue is resolved now. Feel free to test. You can find the release here https://github.com/jonasthewolf/gsn2x/releases.
